README.md

```diff
@@ -230,7 +229,7 @@ make script-Deploy PROTOCOL=compound NETWORK=goerli SMODE=local
 
 ## Questions & Feedback
 
-For any question or feedback you can send an email to [merlin@morpho.xyz](mailto:merlin@morpho.xyz).
+For any questions or feedback, you can send an email to [merlin@morpho.org](mailto:merlin@morpho.org).
```

Comment:
Do we want to keep that or just redirect to the Issues section?

Reply:
IMO it's weird